css/style.css

```diff
@@ -0,0 +1,214 @@
+html {
+    box-sizing: border-box;
+    font-size: 62.5%;
+    scroll-behavior: smooth;
+}
+
+body {
+    font-family: 'Poppins', sans-serif;
```

Comment:
Please import font family to top of your css file to use Poppins font.

Reply:
- [ ] Please give reasons to each comment so the student will not feel that the requested change is unfair :+1: